_includes/tutorials/message-ordering/kafka/code/tutorial-steps/test/dump-log-segments-1.sh

```diff
@@ -0,0 +1,4 @@
+docker-compose exec broker kafka-run-class kafka.tools.DumpLogSegments \
```

Comment:
Instead of `kafka-run-class kafka.tools.DumpLogSegments`, there's a script `kafka-dump-log` which can be used.